pdf_beamtime_server:
  ros__parameters:
    home_angles: [0.0, -1.783732, 0.0, -1.75492, 0.0, 3.14159] #For real robot
    gripper_present: true
